### ISSUE TITLE
Admin group removed from sysmeta of portal

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1442]
Somehow a user was able to remove the admin group from the sys meta, although the admin group is configured as a "hidden" subject in the AppModel. 

Last sysmeta with admin group:

```xml
<ns3:systemMetadata xmlns:ns2="http://ns.dataone.org/service/types/v1" xmlns:ns3="http://ns.dataone.org/service/types/v2.0">
<script/>
<serialVersion>1</serialVersion>
<identifier>urn:uuid:f472a78b-4344-4077-ba88-8d790f36a69c</identifier>
<formatId>https://purl.dataone.org/portals-1.0.0</formatId>
<size>1936</size>
<checksum algorithm="MD5">ee1668c46e87583cb7e6848e81d87f8f</checksum>
<submitter>http://orcid.org/0000-0002-4222-1544</submitter>
<rightsHolder>http://orcid.org/0000-0002-4222-1544</rightsHolder>
<accessPolicy>
<allow>
<subject>CN=arctic-data-admins,DC=dataone,DC=org</subject>
<permission>read</permission>
<permission>write</permission>
<permission>changePermission</permission>
</allow>
</accessPolicy>
<replicationPolicy replicationAllowed="false"/>
<obsoletedBy>urn:uuid:6ed19b94-5268-4842-9eb0-bc582b2d9aea</obsoletedBy>
<archived>false</archived>
<dateUploaded>2020-06-20T13:35:20.867+00:00</dateUploaded>
<dateSysMetadataModified>2020-06-20T13:36:14.875+00:00</dateSysMetadataModified>
<originMemberNode>urn:node:ARCTIC</originMemberNode>
<authoritativeMemberNode>urn:node:ARCTIC</authoritativeMemberNode>
<seriesId>urn:uuid:19d2c5f7-506b-4196-81aa-52a80caebd95</seriesId>
<fileName>SASS.xml</fileName>
</ns3:systemMetadata>
```

Next updated version:

```xml
<ns3:systemMetadata xmlns:ns2="http://ns.dataone.org/service/types/v1" xmlns:ns3="http://ns.dataone.org/service/types/v2.0">
<script/>
<serialVersion>1</serialVersion>
<identifier>urn:uuid:6ed19b94-5268-4842-9eb0-bc582b2d9aea</identifier>
<formatId>https://purl.dataone.org/portals-1.0.0</formatId>
<size>1936</size>
<checksum algorithm="MD5">ee1668c46e87583cb7e6848e81d87f8f</checksum>
<submitter>http://orcid.org/0000-0002-4222-1544</submitter>
<rightsHolder>http://orcid.org/0000-0002-4222-1544</rightsHolder>
<accessPolicy>
<allow>
<subject>Joe Schmoe</subject>
<permission>read</permission>
<permission>write</permission>
<permission>changePermission</permission>
</allow>
</accessPolicy>
<replicationPolicy replicationAllowed="false"/>
<obsoletes>urn:uuid:f472a78b-4344-4077-ba88-8d790f36a69c</obsoletes>
<obsoletedBy>urn:uuid:28e9deec-25ac-41c2-836b-2c65ba303606</obsoletedBy>
<archived>false</archived>
<dateUploaded>2020-06-20T13:36:14.467+00:00</dateUploaded>
<dateSysMetadataModified>2020-06-20T13:36:31.391+00:00</dateSysMetadataModified>
<originMemberNode>urn:node:ARCTIC</originMemberNode>
<authoritativeMemberNode>urn:node:ARCTIC</authoritativeMemberNode>
<seriesId>urn:uuid:19d2c5f7-506b-4196-81aa-52a80caebd95</seriesId>
<fileName>SASS.xml</fileName>
</ns3:systemMetadata>
```

"Joe Schmoe" above was the first and last name of the user. It looks like he may have typed his name in and saved that free-text as the subject? (We may want to restrict to ORCIDs and groups only).

**Comment by laurenwalker** [comment]
After looking into this for a bit, it's not immediately apparent how this happened. But I think this can be avoided through the following improvements:
- [ ] Find original bug and fix
- [ ] Don't allow users to remove themselves as an owner. (This can be configurable). I was able to remove myself as the rightsHolder, which bumped up the admin group to the rightsHolder. If a user is not in that admin group, they won't have ownership over their content anymore. -- This is indirectly related to this issue, but important
- [ ] Consider adding a "alwaysUseAccessPolicy" config that will always add those AccessRules to the system metadata when it is serialized, no matter what.

**Comment by laurenwalker** [comment]
I found the original bug and fixed it in the referenced commit, which was caused by a domino effect of when the AccessPolicyView is first rendered, the rightsHolder is not displayed, because the rightsHolder is only added later during save(). Clicking save and then adding new rules to the policy view messed things up.

I also changed all our theme configs to disable editing rightsHolders in the AccessPolicyView. This should work as a safeguard that people won't remove themselves as an owner. (We may still want to implement my second checkbox above - prevent anyone from removing `changePermission` on themselves).
